x-pack/legacy/plugins/ml/public/application/data_frame_analytics/pages/analytics_management/components/create_analytics_form/create_analytics_form.tsx

```diff
@@ -159,27 +202,43 @@ export const CreateAnalyticsForm: FC<CreateAnalyticsFormProps> = ({ actions, sta
       const indexPattern: IndexPattern = await kibanaContext.indexPatterns.get(
         indexPatternsMap[sourceIndex].value
       );
+      const shouldPrepDepVarOptions =
+        jobType === JOB_TYPES.REGRESSION || jobType === JOB_TYPES.CLASSIFICATION;
 
       if (indexPattern !== undefined) {
         await newJobCapsService.initializeFromIndexPattern(indexPattern);
         // Get fields and filter for supported types for job type
         const { fields } = newJobCapsService;
-        const options: Array<{ label: string }> = [];
+
+        const depVarOptions: Array<{ label: string }> = [];
```

Comment:
Could be replaced with `EuiComboBoxOptionProps`

Reply:
Updated here 5fb02c50cabc18a94c4952adee9d279bea7b54ac